Makefile

```diff
@@ -70,6 +70,7 @@ phylumpush:
 all: oracle
 .PHONY: oracle
 oracle: plugin
+	$(MAKE) api
```

Comment:
My understanding was that adding it as a dependency would make it so that if `api` was built, then `oracle` would be built. What I meant to do here was the opposite - if oracle is being built, rebuild api beforehand. An error I kept seeing was:

> `oracle_test.go:68: unable to initialize mock phylum: SUBSTRATEHCP_FILE not found in environment`

and while I'm unsure this is the simplest/most correct patch, rebuilding `api` seemed to fix that. Usually.

Reply:
Ah, no this should be what you want.  To build oracle when api is built would be the other direction, e.g. `api: oracle`.

Reply:
This didn't seem to actually help, reverted.